### DISCUSSION TITLE
Organise Release parties

### DISCUSSION THREAD

**Discussion by danuw** [Green-Software-Foundation/carbon-aware-sdk#66] [General]
Today we discussed having releases every two months and bug fixes every other month (in between bigger releases).

How about organising release parties and how might we go about it? i.e. Latest release's features owners should be expected to present back, can we host these on youtube...?

(btw marked as Show and Tell, as that is a show and tell idea, but may be the wrong category)

**Comment by yuu1127** [comment]
IMO, uploading on youtube sounds a good idea.
Participants can review the recording and 
others who couldn't attend the meeting can know how is the project going and feel easy to catch up as well.

**Reply by Willmish** [reply]
We definitely need to store the recordings for other people, good call. Having it live as well would be nice though to have some more interaction and immediate feedback.

**Comment by danuw** [comment]
Thank you @yuu1127, I was thinking along the lines of 1 of these 2 options:

Option 1 (aiming for 30 minutes)
- Intro (GSF, join us etc..)
- Carbon Aware SDK context/Intro
- The release and what we covered in it
- Demo / how to get started
- What we are looking into next?
- Q&A

Option 2 (option 2 30-60 minutes)
- Intro (GSF, join us etc..)
- Carbon Aware SDK context
- The release and what we covered in it
- Demo
- What we are looking into next?
- Walkthrough on how to get started for people who want to stay in and a deeper dive
- Q&A

I suppose, next we would need to fix a date (I would say 2 weeks after the official release date for the first one) and then expect one of those 0 to 2 days after the next releases (major or minor)
